docs/examples/Track_Stitcher_Example.py

```diff
@@ -290,12 +279,11 @@
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 # Now that we have a set of tracklets, we can apply the Track Stitching method to stitch tracklets
 # together into tracks. The code in the following cell applies this process using the class
-# `TrackStitcher` and plots the stitched tracks.
-# `TrackStitcher` has a property search_window that enables you to reduce compute time by filtering
-# out track segments that are not within a determined time window. When forward stitching, the
-# associator will consider any track that has a start point that falls within the time window
-# :math:`(t, t + search_window)`. When backward stitching, the associator will consider tracks that
-# have an endpoint within the time window :math:`(t - search_window, t)`.
+# `TrackStitcher` and plots the stitched tracks. `TrackStitcher` has a property 'search_window' that
+# reduces computation time by filtering out track segments that outside a defined time window. When
+# forward stitching, the associator will consider any track that has a start point that falls within
+# the time window :math:`(t, t + search_window)`. When backward stitching, the associator will
```

Comment:
```suggestion
# the time window :math:`(t, t + search\_window)`. When backward stitching, the associator will
```
You need to escape the underscore (I think with `\`) or the W in "window" is made subscript.